File: tests/fixtures/aspnet-comprehensive/Controllers/AuthController.cs
using AspNetComprehensive.Models;
using Microsoft.AspNetCore.Mvc;

namespace AspNetComprehensive.Controllers;

[ApiController]
[Route("api/auth")]
public class AuthController : ControllerBase
{
    [HttpPost("login")]
    public IActionResult Login([FromBody] LoginRequest body) => Ok(new { token = "fake" });

    [HttpPost("logout")]
    public IActionResult Logout() => NoContent();
}

File: tests/fixtures/springboot-comprehensive/src/main/java/com/example/auth/AuthController.java
package com.example.auth;

import org.springframework.web.bind.annotation.*;

@RestController
@RequestMapping("/api/auth")
public class AuthController {

    @PostMapping("/login")
    public String login(@RequestBody @Valid LoginRequest body) { return "fake-token"; }

    @PostMapping("/logout")
    public void logout() {}
}